keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/MissingAppVersionException.kt

```diff
@@ -0,0 +1,7 @@
+package com.netflix.spinnaker.keel.ec2
+
+import com.netflix.spinnaker.kork.exceptions.IntegrationException
+
+class MissingAppVersionException(
+  val resourceId: String
+) : IntegrationException("Resource $resourceId is missing the appVersion, is it present on the AMI?")
```

Comment:
TIL about this `IntegrationException`.
Question: I think the exception message may be a little cryptic if it bubbles up all the way. Can we maybe make that a bit more friendly? Where does the `appVersion` come from in the AMI, is it a tag? If it is, we could use something like "Resource X is missing version information. Is the appVersion tag missing from the AMI?".

Reply:
This should never happen TM, but i have seen this situation maybe once... It'll need us to debug regardless since it requires a bunch of knowledge of the system and our netflix assumptions. It does come from the ami tag, i'll update the exception to be the language you suggested.